crdppf/lib/geometry_functions.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: UTF-8 -*-
+
+from crdppf.models import DBSession
+from crdppf.models import Property, PaperFormats
+
+from sqlalchemy.sql import func
+
+def get_feature_center(idemai):
+    """ Extract a feature centroid regarding its idemai attribute
+    """
+
+    geom = DBSession.query(Property.geom).filter(Property.idemai==idemai). \
```

Comment:
Hmmm... yes... why not...

But how do you except the requested route to answer to clients ? Should we send back an error 500 ?